spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -430,6 +430,52 @@ definitions:
                     - 2: +/- 10 microseconds
                     - 3: < 1 microseconds
 
+ - MSG_SENSOR_AID_EVENT:
+    id: 0xFF09
+    short_desc: Sensor state and update status data
+    desc: >
+      This diagnostic message contains state and update status information for
+      all sensors that are being used by the fusion engine. This message will
+      be generated asynchronously to the solution messages and will be emitted
+      anytime a sensor update is being processed.
+    fields:
+        - time:
```

Comment:
You changed this from `tow` in the design doc to `time`. Which time is it? If it's variable, should we explicitly include a time source flag?

Reply:
I wanted to keep the door for that open and found the `tow` field name confusing e.g. if outputting a CPU local time. We have 32 reserved bits in the `flags` field. I anticipate us using a portion of them in a common way for all sensors, while other bits might have specialised meanings for every sensor. We could use some of the common bits in the future to signal which time we're reporting (CPU local, GNSS, or something else)